.github/workflows/megalinter.yaml

```
@@ -48,6 +48,7 @@ jobs:
       contents: write
       issues: write
       pull-requests: write
+      workflows: write
```

actionlint (1.7.7)

unknown permission scope "workflows". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)